Assets/_BForBoss/_Entities/Scripts/HealthbarViewBehaviour.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Perigon.Entities
+{
+    [RequireComponent(typeof(Image))]
+    public class HealthbarViewBehaviour : MonoBehaviour
+    {
+        private Image _healthbarImage;
+        private ILifeCycle _lifeCycle = null;
+
+        private void Awake()
+        {
+            _healthbarImage = GetComponent<Image>();
+        }
+
+        public void Initialize(ILifeCycle lifeCycle)
+        {
+            _lifeCycle = lifeCycle;
+            
+            
+        }
+
+        private void Update()
+        {
+            _healthbarImage.fillAmount = GetHealthBarAmount();
```

Comment:
Another way to set this up instead of updating this every frame, is for LifeCycle to have a callback when the `CurrentHealth` get's updated

Reply:
but it's your choice if u want to leave it like this.